application/testing/CMakeLists.txt

```diff
@@ -620,3 +620,8 @@ f3d_test(NAME TestBadRefNoOutput DATA cow.vtp ARGS --ref ${CMAKE_SOURCE_DIR}/tes
 
 # Test failure with a bad interaction play file, please do not create a dummy.log
 f3d_test(NAME TestPlayNoFile DATA cow.vtp ARGS --interaction-test-play=${CMAKE_BINARY_DIR}/Testing/Temporary/dummy.log WILL_FAIL)
+
+# Test scan plugins
+if(NOT F3D_MACOS_BUNDLE)
+  f3d_test(NAME TestScanPlugins ARGS --scan-plugins NO_RENDER NO_BASELINE REGEXP " - native")
+endif()
```

Comment:
I think other tests do not make sense in MACOS bundle then ?

Reply:
TestPluginName and TestPluginFullPath ?

Reply:
No these ones are fine. The problem with the bundle is only related to the plugin json files.